infomaniak-build-tools/kDrive-template.html

```diff
@@ -29,6 +29,7 @@ <h2><b>Bug fixes</b></h2>
     <li>Improved the management of access denied errors.</li>
     <li>Improved user experience for sending logs.</li>
     <li>macOS - Improved German translation in installer.</li>
+    <li>macOS - Fixed a bug that could block synchronization when renaming a file with only a case change.</li>
```

Comment:
```suggestion
    <li>macOS - Fixed a bug that could block synchronization when renaming a file or a directory while changing only the type case.</li>
```
Hopefully "type case" (https://en.wikipedia.org/wiki/Type_case) will correctly be translated in  the french word "casse".